src/routes/identity/account/+page.svelte

```diff
@@ -48,7 +48,7 @@
 		<p class="text-lg">This address is used to send and receive tokens with Waku objects</p>
 	</Container>
 	<Container gap={6}>
-		<Textarea value={$walletStore.wallet?.address} disabled label="Account address" rows={2} />
+		<Textarea value={$walletStore.wallet?.address} disabled label="Account address" />
```

Comment:
These likely should not be Textareas but some other components. The reason is that you can not select and copy from disabled textarea.

Reply:
100% agree.